client/src/app/app.component.ts

```diff
@@ -192,6 +194,8 @@ export class AppComponent implements OnInit {
   private async loadPlugins () {
     this.pluginService.initializePlugins()
 
+    setTimeout(() => this.pluginService.initializeCustomModal(this.customModal))
```

Comment:
Put this in the `ngAfterViewInit()` hook instead of using `setTimeout`: https://angular.io/guide/lifecycle-hooks

Reply:
Maybe we should add a hook / event / obervable in `ngAfterViewInit()` too if we want the helper to be available and not being undefined when the plugin is initialized ?